polymesh_schema.json

```diff
@@ -573,6 +574,19 @@
                 "type": "Vec<u32>"
             }
         },
+        "protocolFee": {
+            "computeFee": {
+                "description": "Gets the fee of a chargeable extrinsic operation",
+                "params": [
+                    {
+                        "name": "op",
+                        "type": "ProtocolOp",
+                        "isOptional": false
+                    }
```

Comment:
need to add the `at` as well as you can see in the other rpc definitions

Reply:
Done, if you mean `blockHash`. It was indeed missing.

Reply:
Yeah